packages/meta-harness/tests/test_evaluator_worktree_lifecycle.py

```
@@ -273,3 +281,177 @@ def test_valid_overlay_is_applied_without_error(
+            "config_hash": ev.mh.compute_config_hash(overlay_dir, {}),
+            "config_patch_hash": ev.mh.compute_config_patch_hash(patch),
+        }
+        if mutation == "tamper":
+            patch[0]["value"] = "tampered-model"
+            patch_path.write_text(json.dumps(patch), encoding="utf-8")
+        else:
+            patch_path.unlink()
+        worktree_dir = tmp_path / "worktree-tampered"
+        worktree_dir.mkdir()
+
+        with pytest.raises(ev.EvaluatorStageError, match="hash mismatch|sidecar is missing"):
```

Ruff (0.15.21)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)